### DISCUSSION TITLE
2023 UX/Design strategy

### DISCUSSION THREAD

**Discussion by scottveirs** [orcasound/.github#12] [General]
Met with @UXBrendan and Oya this evening to discuss UX/Design strategy. I'm not sure we stuck with it, but Brendan started us off with a great aspiration for managing the steady stream of UX/design volunteers in 2023: "focusing on quality rather than quantity, and finding a simple management solution that doesn't require extended time and effort, would be best."

Brendan also did a nice job of summarizing our conversation for the rest of the UX management team, including @shaneuxd:

**UX strategy for 2023, there was some clarity about what projects to focus on:**

- Expanding the Professional Marine Scientist persona
  - May include subpersonae
- Researching and designing the OrcaLearn app
- Running research and design projects for the bioacoustic dashboard
- Integrating Acartia ([acartia.io](http://acartia.io/)) into the Orcasound ecosystem
  - Understanding why, how, cadence, and where to implement notifications
  - Beyond hydrophone orca listening events
- Moving forward with the Shipnoise app

My main suggestion regarding a management solution was (mostly my gut feeling) that it could help to move more and more of the UX/design process (e.g. asynchronous collaboration of different teams spread across many time zones) and products (mockups, feature requests, user data analyses) closer to the dev teams by using Github's growing suite of tools. Maybe mockups can live within a repo's `Wiki`, feature requests can be labeled as such within `Issues,` key threads from Slack can be archived and discussed more deeply in `Discusssions,` some project management can migrate (from Trello, Asana, etc) towards `Projects,` and perhaps even repos could be established for versioning key assets (e.g. logos, audio libraries, graphics, email templates) ?

**Comment by UXBrendan** [comment]
Since the meeting referenced in the above post, the UX Team has moved all project management to the [Product](https://github.com/orcasound/product) repo.
